ci/tasks/prepare-e2e-codebuild.yml

```diff
@@ -87,3 +87,18 @@ run:
         }
       }
       EOF
+
+      echo
+      echo "Zap test configuration"
+      cat <<EOF | tee ./run-codebuild-configuration/zap.json
+      {
+        "projectName": "endtoend-tests-test-12",
+        "sourceVersion": "${PAY_CI_VERSION_ID}",
+        "secondarySourcesVersions": {},
+        "environmentVariables": {
+          $SOURCE_REPO_CONFIG
+          "tag_${PROJECT_UNDER_TEST}": "${RELEASE_TAG_UNDER_TEST}",
```

Comment:
You need to pull the latest version, this file has updated and the tag env var isn't PROJECT_UNDER_TEST anymore (thanks to products-ui having a - in the name)

Reply:
Fixed